### ISSUE TITLE
Maintainership of Flathub package

### ISSUE THREAD

**Issue by bbhtt** [hydrusnetwork/hydrus#1385]
### Hydrus version

NA

### Qt major version

Qt 6

### Operating system

Linux (specify distro and version in comments)

### Install method

Third party (AUR, Docker, Chocolatey, etc. Specify in comments)

### Install and OS comments

NA

### Bug description and reproduction

Hello, someone submitted this app on Flathub a while back (https://github.com/hydrusnetwork/hydrus/issues/1117). They since deleted their account, so the package is currently without maintainers and lagging behind in updates.

I opened a PR to update it on Flathub's side https://github.com/flathub/io.github.hydrusnetwork.hydrus/pull/13 but since it has no maintainers, no one except Flathub admins can merge PRs there.

But since admins aren't users/contributors here, it makes everyone hesitant to merge.

Is anyone from upstream willing to become maintainer of the package? It receives a sizable number of installs, so it'd be a shame to EOL it and we can't also keep it behind in updates.

In case the developers here don't have Flathub domain knowledge, I (or someone else) can send PRs to update it and this can even be automated.

So basically maintainership at the very least will entail testing the builds, green lighting them if every thing works and hitting merge.

If anyone from upstream (member/contributor etc.) is interested, please leave a comment https://github.com/flathub/flathub/issues/3693 or ask in #flatpak:matrix.org

Thanks!
